src/org/exist/storage/txn/Txn.java

```diff
@@ -61,10 +61,6 @@ public long getId() {
         return id;
     }
     
-    public void registerLock(final Lock lock, final LockMode lockMode) {
```

Comment:
Yes, however I really did not think anyone outside of the eXist codebase would be using it. Do you want me to re-add it and mark it as deprecated?